src/components/pages/rules/flyouts/ruleEditor/ruleEditor.scss

```diff
@@ -15,4 +15,8 @@
     padding-left: 0px;
     @include rem-fallback(margin-top, 24px);
   }
+
+  .thin-pad-top{
```

Comment:
Stylistic nit. When ever a sass declaration has only one property, the entire declaration goes one line.
`.thin-pad-top { @include rem-fallback(margin-top, 10px); }`

Reply:
Changed.